llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -1665,7 +1665,7 @@ bool LoopVectorizationLegality::isVectorizableEarlyExitLoop() {
 
   // Keep a record of all the exiting blocks.
   SmallVector<const SCEVPredicate *, 4> Predicates;
-  std::optional<std::pair<BasicBlock *, BasicBlock *>> SingleUncountableEdge;
+  BasicBlock *SingleUncountableExitingBlock;
```

Comment:
Could the build issue be due to this uninitialised variable?

Reply:
Ah yes, thanks, fixed. Looks like my local compiler was zero-initializing local variables...